src/Service/Caller/Masa.Contrib.Service.Caller/Internal/CallerDependExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Service.Caller.Internal;
+
+/// <summary>
+/// Caller dependency orchestration
+/// </summary>
+internal static class CallerDependExtensions
+{
+    /// <summary>
+    /// Caller dependency orchestration
+    /// </summary>
+    /// <param name="callerTypes">All Callers that inherit CallerBase</param>
+    /// <returns></returns>
+    /// <exception cref="UserFriendlyException"></exception>
+    public static List<Type> Arrangement(this List<Type> callerTypes)
```

Comment:
What's the mean of Arrangement

Reply:
Sort the implementation of integrated CallerBase, and finally implement Caller nesting